11-kerberos-ticket-policy.rst

```diff
@@ -0,0 +1,387 @@
+service has own Kerberos service and, optionally, alias names as well. Depending
+on who initiates a communication flow, in Kerberos terminology there are client
+and a target (server) principals. Any Kerberos principal can be used in both
+client and server roles, though typically FreeIPA services are used as targets
+and users are used as clients.
+
+A client first authenticates to a Key Distribution Center and obtains a ticket
+granting ticket (TGT). With the ticket granting ticket in possession, the client
+then asks a KDC for a service ticket to the target Kerberos principal. After the
+service ticket was issued by the KDC, the client can initiate its
+application-specific communication to the target application server.
+
+When a KDC issues a ticket, there are few properties that can be controlled with
+the help of a Kerberos policy in FreeIPA. Each ticket has own life time and a
+potential renewal age: a ticket can be renewed before its life time has ended
+but until the renewal age expired.
+
+The combination of Kerberos ticket life time and renewal age altogether
+comprises a **Kerberos ticket policy**. The policy itself is not stored in a single
+place because individual parts of it are applied to different objects and at
+different stages of authentication and authorization processes. Instead, it is
+split into several parts which are associated with the corresponding Kerberos
+principals, whether they are used as clients or as targets.
+
+For users, Kerberos ticket life time and renewal age can be managed with
+Kerberos ticket policy commands described in ``ipa help krbtpolicy`` manual.
+
+If no specific policy is associated with a user, a default one is applied. To
+manage default policy the same ``ipa krbtpolicy-*`` commands are used, without
+explicit user name.
+
+Display the current Kerberos ticket policy::
+
+  [admin@client]$ kinit admin
+  Password for admin@IPADEMO.LOCAL:
+  [admin@client]$ ipa krbtpolicy-show
+    Max life: 86400
+    Max renew: 604800
+
+Reset the policy to the default::
+
+  [admin@client]$ ipa krbtpolicy-reset
+    Max life: 86400
+    Max renew: 604800
+
+Modify the default policy to 8 hours max life, 1-day max renewal::
+
+  [admin@client]$ ipa krbtpolicy-mod --maxlife=28800 --maxrenew=86400
+    Max life: 28800
+    Max renew: 86640
+
+Display effective Kerberos ticket policy for user ``admin``::
+
+  [admin@client]$ ipa krbtpolicy-show admin
+    Max life: 28800
+    Max renew: 86640
+
+Modify per-user policy for user 'admin'::
+
+  [admin@client]$ ipa krbtpolicy-mod admin --maxlife=3600
+    Max life: 3600
+
+Reset per-user policy for user ``admin``::
+
+  [admin@client]$ ipa krbtpolicy-reset admin
+    Max life: 28800
+    Max renew: 86640
+
+Currently (FreeIPA 4.8.4), FreeIPA does not allow to associate a custom Kerberos
+ticket policy with a Kerberos service principal. As result, only default
+Kerberos ticket policy is applicable to service principals.
```

Comment:
to associating -> associating

or "FreeIPA does not allow a Kerberos service principal to have a custom Kerberos ticket policy."